doc/man1/openssl-genrsa.pod.in

```diff
@@ -126,7 +123,7 @@ L<openssl-gendsa(1)>
 
 =head1 HISTORY
 
-This command was deprecated in OpenSSL 3.0.
+This command was erroneously marked as deprecated in OpenSSL 3.0. It is not deprecated.
```

Comment:
I suggest you just remove the whole HISTORY section.

Reply:
@Abdullah-03 Please just remove the section as @kroeckx suggests.

Reply:
Sorry for the delay. I'll remove it.